packages/webapp/pages/cores/index.tsx

```diff
@@ -164,6 +167,18 @@ export const CorePageRenderer = ({
     paddle,
   ]);
 
+  useEffect(() => {
+    if (checkCoresRoleNotNone(user)) {
```

Comment:
for anon user it will be true so it does not redirect as well

Reply:
So if anon user, also redirect?

Reply:
yes, because we don't know their role

Reply:
Added redirect to onboarding if anon, and home if non-anon.